### DISCUSSION TITLE
The Beacon station generated too close to the outpost

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14757] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I played yesterday on a boat :
https://steamcommunity.com/sharedfiles/filedetails/?id=3314435085
The boat is based on the R-29

After the start of the mission, I discovered that the beacon station was so close to the starting outpost that my submarine intersected with it. Because of this, the boat could not move.

The Beacon station was generated too close to the outpost. This led to a collision with the submarine. The boat got stuck and being in the wheelhouse or in the middle of the boat, I was simultaneously inside the beacon station.

To successfully complete the mission, I got into my shuttle, go to the destination outpost and docked the shuttle.

After starting the next game session at the station, I discovered that I no longer had a boat, only a shuttle. Is this a bug or was it intended? I completely lost the boat and all the things in it, only the shuttle remained. In the "purchased boats terminal", there is no Kamaz6_4 anymore, only the shuttle.

Unfortunately, I did not think to make a copy of the save with the collision of the boat and the beacon.
But I saved the log.
The level seed looks strange
Level seed: LocationName.alien.4877LocationName.city.693

Here is the log:

[vanillaSPB_2024-09-30_2118.txt](https://github.com/user-attachments/files/17202847/vanillaSPB_2024-09-30_2118.txt)

### Reproduction steps

I do not exactly how to reproduce first bug.
To reproduce the second bug, it is enough to undock the shuttle from the boat after the start of the mission and dock it to the station, completing the mission on the shuttle. I think so.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
The sub getting lost is intentional: if you leave your submarine in the level and just take a shuttle to the outpost, the sub will be left behind. You can still recover it with the "purchase lost shuttles" option on the outpost engineer or by returning to the level and taking control of it.

The beacon spawn seems to be a bug, but unfortunately I don't think we can reproduce the issue without the save (or by hoping we'll find another level where it happens). :/
